packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,274 @@
+
+  /**
+   * Container styling
+   */
+  let containerStyle: ViewStyle = {
+    width: '100%',
+  }
+
+  if (error) {
+    containerStyle = {
+      ...containerStyle,
+      borderLeftWidth: spacing.vadsSpace2xs,
+      borderColor: theme.vadsColorFormsBorderError,
+      paddingLeft: spacing.vadsSpaceLg,
+    }
+  }
+
+  /**
+   * Pressable styling
+   */
+  const pressableBaseStyle: StyleProp<ViewStyle> = {
+    width: '100%',
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+  }
+
+  const tileStyle: ViewStyle = {
+    ...pressableBaseStyle,
+    borderWidth: 2,
+    borderRadius: spacing.vadsSpace2xs,
+    paddingTop: spacing.vadsSpaceSm,
+    paddingBottom: spacing.vadsSpaceSm,
+    paddingLeft: spacing.vadsSpaceSm,
+    paddingRight: spacing.vadsSpaceMd,
+    borderColor: checked
+      ? theme.vadsColorFormsBorderActive
+      : theme.vadsColorFormsBorderSubtle,
+    backgroundColor: checked
+      ? theme.vadsColorFormsSurfaceActive
+      : theme.vadsColorSurfaceDefault,
+  }
+
+  /**
+   * Header
+   */
+  const headerProps = {
+    'aria-label': headerA11y,
+    style: {
+      // TODO: Replace with typography tokens
+      fontFamily: 'SourceSansPro-Regular',
+      color: theme.vadsColorForegroundDefault,
+      fontSize: 20,
+      lineHeight: 30,
+    },
+  }
+
+  const _header = header && (
+    <>
+      <Text {...headerProps}>{header}</Text>
+      <Spacer size="xs" />
+    </>
+  )
+
+  /**
+   * Hint
+   */
+  const hintProps = {
+    'aria-label': hintA11y,
+    style: {
+      // TODO: Replace with typography tokens
+      fontFamily: 'SourceSansPro-Regular',
+      color: theme.vadsColorForegroundSubtle,
+      fontSize: 16,
+      lineHeight: 22,
+    },
+  }
+
+  const _hint = hint && (
+    <>
+      <Text {...hintProps}>{hint}</Text>
+      <Spacer size="xs" />
+    </>
+  )
+
+  /**
+   * Error
+   */
+  const errorProps = {
+    'aria-label': `${t('error')}: ${errorA11y || error}`,
+    style: {
+      // TODO: Replace with typography tokens
+      fontFamily: 'SourceSansPro-Bold',
+      color: theme.vadsColorForegroundError,
+      fontSize: 16,
+      lineHeight: 22,
+    },
+  }
+
+  const _error = error && (
+    <>
+      <Text {...errorProps}>{error}</Text>
+      <Spacer size="xs" />
+    </>
+  )
+
+  /**
+   * Label
+   */
+  const labelProps = {
+    'aria-label': labelA11y,
+    style: {
+      // TODO: Replace with typography tokens
+      fontFamily: error ? 'SourceSansPro-Bold' : 'SourceSansPro-Regular',
+      color: theme.vadsColorForegroundDefault,
+      fontSize: 20,
+      lineHeight: 30,
+    },
+  }
+
+  const requiredStyle = {
+    color: theme.vadsColorForegroundError,
+  }
+
+  const _label = (
+    <Text {...labelProps}>
+      {label}
+      {required && <Text style={requiredStyle}>{` (*${t('required')})`}</Text>}
+    </Text>
+  )
+
+  /**
+   * Icon
+   */
+  const iconViewStyle: ViewStyle = {
+    paddingTop: 3,
+    paddingBottom: 3,
```

Comment:
Should no longer be needed as the other props center it within the minHeight that matches the font lineHeight
```suggestion

```